src/style.css

```diff
@@ -0,0 +1,104 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 30px 14px 20px;
```

Comment:
center content in you card wiht flex properties, not paddings

Reply:
Centered card content with flex properties